db/model/botAction.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * db/model/botAction.js
+ *
+ * Once an action is intiated from refocus bots need to be able to
+ * check if they have any actions to perform. This table will act
+ * as an action queue for bots.
+ */
+
+const assoc = {};
+const dbErrors = require('../dbErrors');
+const constants = require('../constants');
+const u = require('../helpers/botUtils');
+
+module.exports = function botAction(seq, dataTypes) {
+  const BotAction = seq.define('BotAction', {
+    id: {
+      type: dataTypes.UUID,
+      primaryKey: true,
+      defaultValue: dataTypes.UUIDV4,
+    },
+    pending: {
+      type: dataTypes.BOOLEAN,
+      defaultValue: true,
+      comment: 'Determines if bot action is still active',
```

Comment:
can we use a status enum instead?
e.g. values: "New", "In progress", "Failed", "Done"